pylab/website/tests/test_about_page.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+from django_webtest import WebTest
+from django.contrib.auth.models import User
+
+from pylab.core.models import Event
+
+
+class AboutPageTests(WebTest):
+
+    def setUp(self):
+        self.user = User.objects.create(username='u1')
```

Comment:
At least from the first look it seems, that you don't use this anywhere in this test. If you don't use this, then delete it.